packages/@lwc/engine-core/src/framework/wiring/context.ts

```diff
@@ -54,9 +54,10 @@ export function createContextProviderWithRegister(
                         consumerDisconnectedCallback(consumer);
                     }
                 };
-                setDisconnectedCallback(disconnectCallback);
+                setDisconnectedCallback?.(disconnectCallback);
 
                 consumerConnectedCallback(consumer);
+                return true;
```

Comment:
Please leave a comment describing why this always returns `true`, e.g. the reason why we want to continue traversing up the tree.